engines/sherlock/detection.cpp

```diff
@@ -26,8 +26,8 @@
 #include "sherlock/sherlock.h"
 
 static const PlainGameDescriptor sherlockGames[] = {
-	{ "scalpel", "The Case of the Serrated Scalpel" },
-	{ "rosetattoo", "The Case of the Rose Tattoo" },
+	{ "scalpel", "The Lost Files of Sherlock Holmes" }, //also called The Lost Files of Sherlock Holmes: The Case of the Serrated Scalpel
```

Comment:
For uniformity, let's use "The Lost Files of Sherlock Holmes: The Case of the Serrated Scalpel" in the title, otherwise it'll be confusing

```suggestion
	{ "scalpel", "The Lost Files of Sherlock Holmes: The Case of the Serrated Scalpel" },
```